newIDE/app/src/MainFrame/index.js

```diff
@@ -3035,15 +3042,23 @@ const MainFrame = (props: Props) => {
                     onOpenPreferences: () => openPreferencesDialog(true),
                     onOpenAbout: () => openAboutDialog(true),
                     selectInAppTutorial: selectInAppTutorial,
-                    onLoadEventsFunctionsExtensions: () =>
-                      eventsFunctionsExtensionsState.loadProjectEventsFunctionsExtensions(
+                    onLoadEventsFunctionsExtensions: () => {
+                      if (isProjectClosed) {
+                        return Promise.resolve();
+                      }
```

Comment:
Same, if currentProject === null is equivalent to isProjectClosed, this is good already and we should strive to keep this.